account_analytic_analysis_recurring/account_analytic_analysis_recurring.py

```diff
@@ -123,6 +123,16 @@ class AccountAnalyticAccount(orm.Model):
         'recurring_interval': fields.integer(
             'Repeat Every', help="Repeat every (Days/Week/Month/Year)"),
         'recurring_next_date': fields.date('Date of Next Invoice'),
+        'state': fields.selection(
+            [('template', 'Template'),
+             ('draft', 'New'),
+             ('open', 'In Progress'),
+             ('pending', 'To Renew'),
+             ('overdue', 'Overdue'),
+             ('close', 'Closed'),
+             ('cancelled', 'Cancelled')], 'Status', required=True,
+            track_visibility='onchange'
```

Comment:
It would be nice if you could add comments identifying what this redefinition changes from the standard.